lbz/resource.py

```diff
@@ -207,6 +207,8 @@ def get_pagination(self, total_items: int, limit: int, offset: int) -> dict:
 
     @property
     def _pagination_uri(self) -> str:
-        if query_params := copy_without_keys(self.request.query_params, "offset", "limit"):
-            return f"{self.urn}?{urlencode(query_params)}&offset={{offset}}&limit={{limit}}"
+        if query_params := self.request.query_params.original_items("offset", "limit"):
+            # TODO: Make MultiDict be a Mapping not a object to remove type ignore below
```

Comment:
This TODO is no longer valid 😉  But to satisfy `mypy` you can change the type of `query_params`:
```python
def original_items(...) -> Sequence[Tuple[Any, Any]]:
    ...
```